src/python/pants/backend/python/tasks/isort_run.py

```diff
@@ -50,6 +51,19 @@ def execute(self):
 
       isort = self.context.products.get_data(IsortPrep.tool_instance_cls)
       isort_subsystem = IsortPrep.tool_subsystem_cls.global_instance()
+      deprecated_conditional(
+        lambda: self.get_passthru_args(),
+        removal_version='1.26.0.dev3',
+        entity_description='Using the old style of passthrough args for isort',
+        hint_message="You passed arguments to isort through either the "
```

Comment:
Maybe this message is something that could be templated? It's a weird case where we want each of these to be slightly different and templating might not be worth it if it's going away in two months, but usually I'd like this templated.

Reply:
I would normally agree, but because this is temporary code that goes away in 1.26.0.dev3 I think we should keep it inlined for simplicity.